Core/source/platform/require.ts

```diff
@@ -2,20 +2,6 @@
 
 'use strict';
 
-/// <reference path="../core/types/openpgp.d.ts" />
-
-export const requireOpenpgp = (): typeof OpenPGP => {
-  // @ts-ignore;
-  if (typeof openpgp !== 'undefined') {
-    // @ts-ignore;
-    return openpgp; // self-contained node-mobile
-  }
-  // running tests on a desktop os node instance
-  // making the require semi-dynamic to surpress Webpack warnings/errors. This line does not rely on webpack at all
-  // if this was webpack, then the `openpgp` variable would be already set, and it would never get down here
-  return require(`${'../../../source/lib/openpgp'}`); // points to flowcrypt-mobile-core/source/lib/openpgp.js
-};
-
```

Comment:
We may still need to do some magic later for it to work within iOS. Hopefully, with the new import mechanism, less magic than before - that is yet to be tested by building this into the iOS assets.

Reply:
CI tests, even those are in the Swift part - are passing, so I assume it just works.

Reply:
They pass because they use already built javascript from before - your changes haven't been updated to the app yet. The built product it's using: https://github.com/FlowCrypt/flowcrypt-ios/blob/master/FlowCrypt/Resources/flowcrypt-ios-prod.js.txt

You can run `npm run-script deploy` which will rebuild the code and replace that file. If you commit after that, Swift/Appium tests will certainly break.

But within this PR, we don't have to do that yet. Let's focus on getting TypeScript pass its own tests + cleaning up the PR as is, and then look into making it work for the app in another PR.

Reply:
ok